src/wovnio/wovnphp/Utils.php

```diff
@@ -8,7 +8,14 @@ class Utils
     // will return the store and headers objects
     public static function getStoreAndHeaders(&$env)
     {
-        $file = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';
+        if (isset($env['WOVN_CONFIG'])) {
```

Comment:
If user set `/path/to/wovn.json` into `$_SERVER['WOVN_CONFIG']`, you don't need to change around here.
Just `$file = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';`  works.

Reply:
When I initially wrote this I thought of it as having two sources of settings, one from $env and one from $_SERVER. And after fixing $_SERVER to $env I forgot to remove the duplicate fragments. Sorry about that, not at my finest today.